tao_compiler/mlir/disc/transforms/codegen_utils.h

```diff
@@ -31,7 +31,12 @@ limitations under the License.
 namespace mlir {
 namespace disc_ral {
 
+#if TENSORFLOW_USE_ROCM
 constexpr int kWarpSize = 32;
```

Comment:
The warp size of ROCM is the same as CUDA here?

Reply:
Warp size should be 64 in ROCm, but modifying to 64 here naively will cause some further incompatible issue in the DISC CodeGen process. We will revisit this number setting later. Leaving it as 32 now can make current CodeGen okay.